test/integration/db/db_tests.dart

```diff
@@ -458,183 +458,186 @@ runTests(db.DatastoreDB store, String namespace) {
           ..addAll(users)
           ..addAll(expandoPersons);
       var allKeys = allInserts.map((model) => model.key).toList();
+      List<db.Key> userKeys = users.map((model) => model.key).toList();
+      List<db.Key> expandoPersonsKeys = expandoPersons.map((model) => model.key).toList();
       return store.commit(inserts: allInserts).then((_) {
-        return waitUntilEntitiesReady(store, allKeys, partition).then((_) {
-          var tests = [
-            // Queries for [Person] return no results, we only have [User]
-            // objects.
-            () {
-              return store.query<Person>(partition: partition).run().toList()
-                  .then((List<db.Model> models) {
-                compareModels([], models);
-              });
-            },
-
-            // All users query
-            () {
-              return store.query<User>(partition: partition).run().toList()
-                  .then((List<db.Model> models) {
-                compareModels(users, models, anyOrder: true);
-              });
-            },
-
-            // Sorted query
-            () async {
-              var query = store.query<User>(partition: partition)
+        return waitUntilEntitiesReady<User>(store, userKeys, partition).then((_) {
+          return waitUntilEntitiesReady<ExpandoPerson>(store, expandoPersonsKeys, partition).then((_) {
```

Comment:
You could consider using an `async` block here:
```dart
await store.commit(inserts: allInserts);
await waitUntilEntitiesReady<User>(...);
await waitUntilEntitiesReady<ExpandoPerson>(...);
```

Reply:
done